config/routes.rb

```diff
@@ -30,6 +30,7 @@ def self.matches?(request)
       "#{ScihistDigicoll::Env.lookup!("app_url_base")}/collections/#{ScihistDigicoll::Env.lookup!("oral_history_collection_id")}?q=#{path_params[:query]}"
     }
 
+    get '/oral-histories/projects', to: redirect { 'https://sciencehistory.org/oral-history-projects' }
```

Comment:
Cool cool. Oh right, i get it now -- this can't be in our existing yml because our existing yml assumes internal link. 

OK, fine, keep it simple, it's fine!

In this simple case with just string literal though, the redirect can be written as:

    get '/oral-histories/projects', to: redirect('https://sciencehistory.org/oral-history-projects' )

param, not a block. 

https://guides.rubyonrails.org/routing.html#redirection